recipes/lenskit/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "lenskit" %}
+{% set version = "0.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 985b7b545dfbed7eb939014f938e685332a40614265c4553fa4bb6d63b43a46e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py install"
```

Comment:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

Reply:
I have made it use `pip`. There still is a manual invocation of `setup.py`, but just to build the helper library. It isn't auto-built during `pip install`. The helper library is useless outside of Conda-based installations (it's only for working with MKL), so I haven't put effort in the upstream package into debugging why pip doesn't build it.